go.mod

```diff
@@ -240,7 +240,7 @@ replace (
 	// Use ethermint fork that respects min-gas-price with NoBaseFee true and london enabled, and includes eip712 support
 	// Tracking kava-labs/etheremint master branch
 	// TODO: Tag before release
-	github.com/evmos/ethermint => github.com/kava-labs/ethermint v0.21.1-0.20240802224012-586960857184
+	github.com/evmos/ethermint => github.com/kava-labs/ethermint v0.21.1-0.20240919184235-65384e03c3e7
```

Comment:
does this have some precompile things that `v0.21.0-kava-v27.0` does not?

Reply:
hmm yes this commit hash version is from master that includes some precompile work -- Will close this PR for #2044 since that one also includes these same ante tests and additional e2e changes, then backport onto `master` with this ethermint version